recipe/meta.yaml

```diff
@@ -24,10 +24,6 @@ source:
       - patches/0013-Add-support-for-_CONDA_PYTHON_SYSCONFIGDATA_NAME-if-.patch
       - patches/0014-Fix-find_library-so-that-it-looks-in-sys.prefix-lib-.patch
       - patches/0015-Disable-new-dtags-in-unixccompiler.py.patch
-      # AppVeyor does not have the 10.0.16299 SDK installed, leave as is
-      # This is actually something else now
-      #- patches/0016-Switch-from-WinSDK-10.0.17134.0-to-10.0.16299.0.patch
-      - winsdk.patch  # [win]
```

Comment:
I'm fairly certain that this will be needed, but lets let AppVeyor run and see for sure.

Reply:
I'll restore it if we need but, I may be confused b/c I did multiple tests early this morning, I think it passed on my AppVeyor account without it.

Reply:
And of course you are right :grimacing: I looked at the wrong log:

https://ci.appveyor.com/project/ocefpaf/python-feedstock/builds/20266954